dice-roller/package.json

```
@@ -10,27 +10,28 @@
 		"ci:test": "start-server-and-test start:server 7070 ci:test:jest",
 		"ci:test:jest": "cross-env NODE_OPTIONS=--experimental-vm-modules jest --ci --reporters=default --reporters=jest-junit",
 		"compile": "tsc -b",
-		"dev": "cross-env FLUID_CLIENT='local' webpack-dev-server",
+		"dev": "cross-env webpack-dev-server",
```

? Or is local the default behavior?
```suggestion
		"dev": "npm run dev:local",
```

---

Yup, if FLUID_CLIENT not azure, use local is the default code behavior